website/pages/docs/platform/k8s/helm/examples/standalone-tls.mdx

```diff
@@ -187,23 +139,19 @@ TMPDIR=/tmp
 
 
 
-### 3. Helm Configuration
+# Helm Configuration
```

Comment:
Is this the same example from yesterday? Look like the header got clobbered?
```suggestion
## 3. Helm Configuration
```

Reply:
Actually this got rearranged so this is the level I wanted it at here. Since this is a different file from the other examples it seemed to make sense to put this at a top level header.

Reply:
Update: I re-copied the example from the current version and ended up keeping it as `## 3. Helm Configuration`